titration/utils/ui_state/user_value/buffer_ph.py

```diff
@@ -0,0 +1,31 @@
+"""
+The file for the BufferPH class
+"""
+
+from titration.utils.ui_state.user_value.user_value import UserValue
+from titration.utils import lcd_interface
+
+
+class BufferPH(UserValue):
+    """
+    This is a class for the BufferPH state of the photometer
+
+    Attributes:
+        photometer (Photometer object): the photometer is used to move through the state machine
+        previous_state (UIState object): the previous_state is used to return the last visited state
+        substate (int): the substate is used to keep track of substate of the UIState
+        string (string): the string is used to hold the user input
+    """
+
+    def save_value(self):
```

Comment:
How about adding a value parameter? Like `save_value(self, value)`. It would be nice to have all the information you need in this view of the class. This way a developer does not need to look at `UserValue` and determine which variable has its final value.

Reply:
This change has been implemented